### PR TITLE
Support virtual folders in the resource/:id/items endpoint.

### PR DESCRIPTION
**Pull request by manthey**

### CODE REVIEW

**Comment by manthey** [comment]
@dgutman This adds support for virtual folders.  I'll add tests soon.

File: server/__init__.py [hunk 1]
```diff
@@ -17,13 +17,32 @@
 #  limitations under the License.
 ###############################################################################
 
+from girder import events
+from girder.api import access
+from girder.api import rest as restUtil
```
Comment (zachmullen) [inline server/__init__.py]: You could save an alias by doing `from girder.api.rest import boundHandler`

**Comment by manthey** [comment]
@zachmullen I made the change you suggested, if you'd care to review it again.

**Comment by manthey** [comment]
Thanks.

**Pull request merged**